1-hello-world/app.js

```diff
@@ -25,13 +25,16 @@ app.get('/', function(req, res) {
 });
 // [END hello_world]
 
-
-// [START server]
-// Start the server
-var server = app.listen(process.env.PORT || 8080, function () {
-  var host = server.address().address;
-  var port = server.address().port;
-
-  console.log('App listening at http://%s:%s', host, port);
-});
-// [END server]
+if (module === require.main) {
```

Comment:
Agreed, but I lean towards keeping the unit tests because they're not just for us, but for also for the user. e.g. here's how you do Node.js + pubsub + logging + _unit tests_ + etc. on GAE.